### ISSUE TITLE
How set new user is_active  False?

### ISSUE THREAD

**Issue by Odin04ka** [Tivix/django-rest-auth#499]
Hi all. Can u help me how create new user with is_active = False? And set is to True when he is come to verification email?

**Comment by Odin04ka** [comment]
REST_AUTH_REGISTER_SERIALIZERS = {
    'REGISTER_SERIALIZER': 'users.serializers.MyRegisterSerializer',
}
and next step set is_active to False. But when u will do that. emails with verification code don't be sended,

**Comment by rpkilby** [comment]
This portion of activation/verification process is handled by allauth, and it doesn't look like there is a trivial way of sending verification emails to inactive users. You'd need to override the call to `complete_signup` here:

https://github.com/Tivix/django-rest-auth/blob/624ad01afbc86fa15b4e652406f3bdcd01f36e00/rest_auth/registration/views.py#L72-L82

`complete_signup` calls `perform_login`, which makes the final check to `is_active` [here](https://github.com/pennersr/django-allauth/blob/cc8a15a1b4bfa6c282109ba2f0f9a36a17625508/allauth/account/utils.py#L130-L131).

In short, you're probably better off not trying to register new users as inactive.
